client/me/help/help-contact/index.jsx

```diff
@@ -164,6 +165,11 @@ const HelpContact = React.createClass( {
 		this.clearSavedContactForm();
 	},
 
+	submitDirectlyQuestion: function( contactForm ) {
+		// TODO: open Directly form here
+		return contactForm;
```

Comment:
I know this `return` isn't used by anything but is it here just to keep the `contactForm` variable from throwing the "This is an unused variable" lint warning? if so then maybe add a comment so that we don't forget to remove it.